types/seat/wlr_seat_keyboard.c

```diff
@@ -330,6 +334,9 @@ static void seat_client_send_keymap(struct wlr_seat_client *client,
 		return;
 	}
 
+	int keymap_fd = -1;
+	void *ptr = NULL;
```

Comment:
These can be defined when needed

Reply:
I'd rather initialize variables on declaration; I'll remove it if you prefer that.

Reply:
No, I mean these can be declared later, on lines 347 and 353.

Reply:
Ah, got it. I thought it would throw "already declared" errors; the more you know.